### PR TITLE
edits for specifics, new highlights

### PR DESCRIPTION
**Pull request by stationaery**

### CODE REVIEW

File: call-for-submissions.md [hunk 1]
```diff
@@ -69,18 +68,16 @@ Here are some examples of content we typically reject:
...
-Here’s how to start the process:
+<mark>Here’s how to start the process:</mark>
 
 1. Email us your rough ideas or pitches to [twcnewsletter@protonmail.com](mailto:twcnewsletter@protonmail.com)!
 1. We’ll reach out to start a conversation.
 1. Then, we’ll help you get a first draft together, usually in a phone call.
 1. We’ll also discuss and decide with you how to handle anonymity and other security considerations.
-1. Depending on the story and strategy, we’ll move it fast or to give it more time.
-1. Finally, we publish the piece! Our newsletters currently comes out every other Friday.
+1. Depending on the story and strategy, we’ll move it fast or to give it more time. Our newsletters currently comes out every other Friday.
```
Comment (jessesquires) [inline call-for-submissions.md:77]: "comes" --> "come"

```suggestion
1. Depending on the story and strategy, we’ll move it fast or to give it more time. Our newsletters currently come out every other Friday.
```

**Comment by jessesquires** [comment]
@stationaery it looks like this will probably conflict with #121 -- do you wanna address and merge that one first?

**Comment by stationaery** [comment]
I wondered where that PR went. In any case, just closed it.

**Comment by jessesquires** [comment]
@stationaery commented on the other PR, but closing != merging... 🤔 

What was your intent?

**Comment by anaulin** [comment]
I've rebased this PR on top of the changes in the previous PR, which I've merged (since that seemed Danny's intent, and the content seemed fine). This is how the changes in this PR look like right now locally:

localhost_4000_call-for-submissions_

**Comment by stationaery** [comment]
LGTM~!

**Review by anaulin — APPROVED** [review]

**Pull request merged**